src/app/pages/common/entity/entity-form/models/field-config.interface.ts

```diff
@@ -23,19 +23,13 @@ export interface InputUnitConfig {
 
 export interface BaseFieldConfig<P = any> {
```

Comment:
We can update `FieldConfig<P = any>` definition not to include `BaseFieldConfig<P>` anymore, since people should be using narrowed down definitions.